sound/soc/sof/sof-priv.h

```diff
@@ -277,6 +277,7 @@ struct snd_sof_pcm_stream {
 	struct sof_ipc_stream_posn posn;
 	struct snd_pcm_substream *substream;
 	struct work_struct period_elapsed_work;
+	int period_elapsed_work_initialized;
```

Comment:
I'd rather use the "bool" type here, especially since you use "true" and "false" to set it.

Reply:
checkpatch.pl yells when we use boolean in structures. I am tempted to make the change though since there is no risk of alignment issues here,

Reply:
@lyakh @plbossart We did a job to remove the bool in struct before. https://github.com/thesofproject/linux/pull/270